study-builder/studies/rgp/enrollment.conf

```diff
@@ -8535,133 +746,26 @@
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.secure_yes} },
-                        { "language": "es", "text": ${i18n.es.secure_yes} }
-                      ]
-                    }
-                  ]
-                },
-                "detailLabelTemplate": null,
-                "allowDetails": false,
-                "exclusive": false
-              },
-              {
-                "stableId": "participate_yes",
-                "optionLabelTemplate": {
-                  "templateType": "TEXT",
-                  "templateText": "$participate_yes",
-                  "variables": [
-                    {
-                      "name": "participate_yes",
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.participate_yes} },
-                        { "language": "es", "text": ${i18n.es.participate_yes} }
-                      ]
-                    }
-                  ]
-                },
-                "detailLabelTemplate": null,
-                "allowDetails": false,
-                "exclusive": false
-              },
-              {
-                "stableId": "delete_yes",
-                "optionLabelTemplate": {
-                  "templateType": "TEXT",
-                  "templateText": "$delete_yes",
-                  "variables": [
-                    {
-                      "name": "delete_yes",
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.delete_yes} },
-                        { "language": "es", "text": ${i18n.es.delete_yes} }
-                      ]
-                    }
-                  ]
-                },
-                "detailLabelTemplate": null,
-                "allowDetails": false,
-                "exclusive": false
-              }
-            ],
-            "validations": [
-              {
-                "ruleType": "NUM_OPTIONS_SELECTED",
-                "minSelections": 3,
-                "maxSelections": 3,
-                "hintTemplate": {
-                  "templateType": "TEXT",
-                  "templateText": "$secure_required",
-                  "variables": [
-                    {
-                      "name": "secure_required",
-                      "translations": [
-                        { "language": "en", "text": ${i18n.en.conditions_required} },
-                        { "language": "es", "text": ${i18n.es.conditions_required} }
-                      ]
-                    }
-                  ]
-                }
-              }
-            ]
-          }
+          "question": ${_questions.agreement}
+        },
+
+        //Questions no longer used
+        {
+          "blockType": "QUESTION",
+          "shownExpr": null,
+          "question": ${_questions.ethnicity_categories}
+        },
+        {
+          "blockType": "QUESTION",
```

Comment:
I'm not sure where this question went, but similar to the above, I would consider putting this at the top of the third page (I believe the third page questions were added when this one was remove, but I couldn't swear to it) or putting it above the agreement checkboxes (just because logically I think it makes sense to have them last).  I could be convinced to agree with any of these locations, though; there isn't really a right answer given that this version of the survey never really existed.

Reply:
> given that this version of the survey never really existed

Interesting note...

---

So the way I went about it with these two questions is by marking them `isDeprecated`. What this flag does is to indicate that this question is no longer used. It exists in the question definition, but won't be displayed in any way to the user. This is nice so that there is a place we can migrate data to for these two questions, but users won't see them at all, and we don't really need to fiddle with versioning here since they're already hidden. We have actually used this feature when migrating one of the other studies, if I recall correctly.

As you have pointed out, looking at the nature of these two questions, it appears to me that they're a variation of the `ethnicity` question and the set of "biological family" questions we currently have. So I figured we can just hide these, even for participants with v1 of the survey. Does this sound good, or should I approach this differently?

Reply:
Leaving this question here for now, unless you have a strong opinion here.

Reply:
That seems reasonable to me